client/src/containers/approver/proposals/TableList.css

```diff
@@ -0,0 +1,14 @@
+/* Copyright 2019 Contributors to Hyperledger Sawtooth
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+----------------------------------------------------------------------------- */
```

Comment:
This file appears to be empty is it needed?

Reply:
We include a stylesheet for every new component as a practice for consistency. Can be changed if desired.